lib/pbench/server/cache_manager.py

```diff
@@ -808,16 +798,23 @@ def get_inventory(self, path: str) -> Optional[JSONOBJECT]:
             Dictionary with file info and file stream
         """
         if not path:
-            info = {
+            return {
                 "name": self.tarball_path.name,
                 "type": CacheType.FILE,
                 "stream": Inventory(self.tarball_path.open("rb")),
             }
         else:
```

Comment:
I was thinking that you would set `path` and define `type` and `stream` here, so that the return at line 817 would do it all.

If you're going to `return` here, then you don't need the `else` at line 806 and you can save a level of indentation....

Reply:
True, but not very important.